File: score-of-a-string.ts
{
    function scoreOfString(s: string): number {
        let c = 0;

        for(let i = 1; i < s.length; i++) {
            c += Math.abs(s[i - 1].charCodeAt(0) - s[i].charCodeAt(0))
        }

        return c
    };

    console.log(scoreOfString('hello'))
}

File: score-of-a-string.rs
pub fn score_of_string(s: String) -> i32 {
    let mut c = 0;

    for i in 0..s.len() - 1 {
        c += (s.as_bytes()[i] as i32 - s.as_bytes()[i + 1] as i32).abs();
    }

    c
}